packages/workers-shared/package.json

```diff
@@ -33,7 +33,7 @@
 		"clean": "rimraf dist",
 		"dev": "pnpm run clean && concurrently -n bundle:asset-worker,bundle:router-worker -c blue,magenta \"pnpm run bundle:asset-worker --watch\" \"pnpm run bundle:router-worker --watch\"",
 		"test": "vitest",
-		"test:ci": "pnpm run test"
+		"test:ci": "pnpm run test run"
```

Comment:
mistake?

Reply:
This is not a mistake as such because the original script is wrong; not adding `run` causes local runs of `test:ci` to hang since this one goes into interactive mode.